go/porcelain/deploy.go

```diff
@@ -344,12 +345,14 @@ func (n *Netlify) DoDeploy(ctx context.Context, options *DeployOptions, deploy *
 		return deploy, nil
 	}
 
-	if err := n.uploadFiles(ctx, deploy, options.files, options.Observer, fileUpload, options.UploadTimeout); err != nil {
+	skipRetry := options.SkipRetry || false
```

Comment:
You don't need `|| false` here. By saying that `options.SkipRetry` is a boolean, it'll get initialised with [the zero value for that type](https://go.dev/tour/basics/12), which is `false`. So you can pass it directly to `uploadFiles`.